buildSrc/version.properties

```diff
@@ -49,9 +49,9 @@ bouncycastle=1.70
 randomizedrunner  = 2.7.1
 junit             = 4.13.2
 hamcrest          = 2.1
-mockito           = 5.1.0
+mockito           = 5.2.0
 objenesis         = 3.2
-bytebuddy         = 1.12.22
+bytebuddy         = 1.14.2
```

Comment:
Do we need changelog entries?

Reply:
Mentioned it in description, skipping changelog since those are only test dependencies, wdyt?

Reply:
Sounds good.